setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import setup
+
+
+with open('cherry_picker/readme.rst') as f:
+    long_description = f.read()
+
+
+with open('cherry_picker/requirements.txt') as f:
+    requirements = [l.strip() for l in f.read().split('\n')]
+
+
+setup(
+    name='cherry_picker',
+    description='backport cpython changes from master to maintenance branches',
+    long_description=long_description,
+    license='Apache',
+    url='https://github.com/python/core-workflow',
+    author='Mariatta Wijaya',
+    author_email='mariatta.wijaya@gmail.com',
```

Comment:
You may want to put `core-workflow@gmail.com` as your address here, so folks know to contact the list with any questions, rather than contacting you directly.